### ISSUE TITLE
file window not opening

### ISSUE THREAD

**Issue by geodeveloper** [NoelOConnell/quill-image-uploader#3]
Hi,

I'm trying to use this module but for some reason the File Window is not opening:
- OS: Mac OS 10.14.2
- Browser: Chrome 71.0

As far as I can debug, I think it has something to do with this code:
```javascript
document.body.appendChild(this.fileHolder);

window.requestAnimationFrame(() => {
    this.fileHolder.click(); // <-- this is not triggering
    window.requestAnimationFrame(() => {
        document.body.removeChild(this.fileHolder);
    });
});
```
If I remove the `requestAnimationFrame` like this:
```javascript
document.body.appendChild(this.fileHolder);
this.fileHolder.click();
```
now it works correctly, the file window shows-up. Is there any reason why `requestAnimationFrame` is preventing to show the File Window? Thanks.

**Comment by NoelOConnell** [comment]
Hi,

The `requestAnimationFrame` block was added in the last release to try and solve #2 
Could you try downgrading to version 1.0.0 and confirm if that version works?

I believe the browser might be blocking the click event in the callback since it did not happen by a user input.

**Comment by t3chnoboy** [comment]
I'm having the same issue with 1.0.1
1.0.0 works fine

**Comment by NoelOConnell** [comment]
@t3chnoboy Thanks for the information.

I have released a 1.0.2 version now.
Tested in Latest Chrome, Firefox, Edge and IE 11

It resolves the issue of the file window not opening in Chrome. Since the `click` event was being triggered in the `requestAnimationFrame`, it was consider outside of a user input event. This is to stop sites being able to show a file window without input from the user.

**Comment by t3chnoboy** [comment]
Thanks for the quick fix! 
1.0.2 works fine with the latest Safari (12.0.2)
